code/modules/hydroponics/grown/nettle.dm

```diff
@@ -25,7 +25,7 @@
 	yield = 2
 	genes = list(/datum/plant_gene/trait/repeated_harvest, /datum/plant_gene/trait/plant_type/weed_hardy, /datum/plant_gene/trait/stinging)
 	mutatelist = list()
-	reagents_add = list(/datum/reagent/toxin/acid/fluacid = 0.25)
+	reagents_add = list(/datum/reagent/toxin/acid/fluacid = 0.12)
```

Comment:
Completely unrelated balance change has no business being part of a feature PR.

Reply:
the feature without rebalancing works like a revert of the previous nerf to the acid because it was 50% before, and adding this trait to nettle will make it back to 50%.
Chems being richer is fine anyway because most of the existing botany chems are all already too much, while a botanist shouldn't throw a single plant with a lot of acid in a short period.
* 13u is the possible maximum without the trait.
* 12u is enough to remove your jumpsuit.
* 26u is enough to remove your captain's hat, and PDA


I mean, a small acid with a single throw is fine, but it shouldn't be too much.